Makefile

```diff
@@ -13,11 +13,19 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+# The goals assume that all Python package dependencies are
+# available, e.g. via a virtualenv.
+
+# Following help file tip from https://stackoverflow.com/questions/8889035/how-to-document-a-makefile
+.PHONEY: help
```

Comment:
```suggestion
.PHONY: help
```

Reply:
Fixed in [b99e5e6](https://github.com/eiffel-community/eiffel/pull/366/commits/b99e5e6dda93ef189a4c5b4ee566cf9db5dcdb00)